Source/Orts.Formats.Msts/Traveller.cs

```diff
@@ -220,7 +220,7 @@ private Traveller(TrackVectorNode startTrackNode, in WorldLocation location, boo
                         throw new InvalidDataException($"Track node {startTrackNode.UiD} could not be found in the track database.");
                     else
                     {
-                        throw new MissingTrackNodeException();
+                        throw new MissingTrackNodeException("Missing Track Node at "+location.TileX+" "+location.TileZ+"!");
```

Comment:
would appreciate formatting, but else seems a good adddition!
`throw new MissingTrackNodeException("Missing Track Node at " + location.TileX + " "+location.TileZ + "!");`
or even better
`throw new MissingTrackNodeException($"Missing Track Node at {location.TileX} {location.TileZ}!");`

Reply:
Thanks for the fast reaction! This already helped me debugging my issue. Formatting looks good. I used your ORTS version because my route is too large for the official one (tdb > 200MB lol).

Reply:
feel free to apply the formatting, and then merge the PR if there's no other change, highly appreciated!!

Are you using the Simulator runtime to run activities, have you also looked at our new "Trackviewer" Toolbox? Would be great to hear more evidence and test cases, as this is a whole new development with focus on performance for larger routes, the simulator runtime is not yet quite on par with all these optimizations, so any contribution is welcome!

Reply:
I would say, we can use the
`throw new MissingTrackNodeException($"Missing Track Node at {location.TileX} {location.TileZ}!");`
one and merge the PR.

Unfortunately I have some trouble to find a working OR working smoothly with my route without stuttering and simulating my timetable correctly. ORTS-MG runs very smoothly, but my timetable doesn't work and it is not rendering some rails (mostly dynatracks). But that's the wrong point here. May I open some new issues for this?

If I find some time, I can test the new Trackviewer more deeply.

Reply:
1. could you please apply the formatting change, update the PR and eventually merge, thanks!
2. Appreciate any feedback as well bug reports - due to the overall complexity, and waste amount of changes in this version, it's almost impossible to test all code paths, and some changes may cause issues not being detected in a long time. Please raise them, ideally as separate issues. Attaching log files, screenshots etc, or link to downloadable content, to help understanding the context, would be highly encouraged.
3. For minor issues or questions, also welcome to use the discussions board, or the gitter IM channel.